CONTRIBUTING.md

```diff
@@ -37,20 +35,8 @@ generate your own `lrgrammar.rs` parser. That's how you do it:
 $ cargo build -p lalrpop # --release
 ```
 
-Now you need to tell cargo that you're using `lalrpop/src/parser/lrgrammar.lalrpop` to generate
-a new `lrgrammar.rs` inside `{CARGO_OUT_DIR}` and use it for the compilation:
-
-```sh
-$ cargo build/test -p lalrpop --features=test
-# or
-$ cargo build/test -p lalrpop --all-features
-```
-
-When this flag is passed, cargo uses not `lalrpop/src/parser/lrgrammar.rs` but the newly generated
-`lrgrammar.rs` instead (generating it if needed). From now on you use this command.
-
-Once you're done with your work, all the tests are passed, and you are ready to finally commit
-you changes run
+Once you're done with your work, make sure that you are running against an
+updated version of the grammar.
```

Comment:
Maybe mention the update_grammar.sh script here as well?

Reply:
That is the next line, it just doesn't appear in the diff?

Reply:
Oh, yep.  My bad.